### ISSUE TITLE
Adding a help page, and a prominent button that leads to it

### ISSUE THREAD

**Issue by zksmk** [LemmyNet/lemmy-ui#201]
### Is your proposal related to a problem?

New users on Lemmy have a hard time figuring out on their own they can access and interact with other instances' communities from their own home instance. This feature is not easily discoverable.  And once they do learn of this aspect of federation, they don't intuitively know how to do it.

### Describe the solution you'd like

While many users join lemmy from join.lemmy.ml and that "on-boarding" process  explains federation sufficiently enough for that step, many users are still confused once on the websites themselves. Adding a small “What is Lemmy and how does it work” tutorial page and having it linked prominently under the signup/login button on the front page could fix it. Could be just a “Help?” button. It could lead to a simple non-overwhelming page that explains the federation concept in layman terms. I prefer the simple [“public email”](https://lemmy.ml/post/52719/comment/34235) comparison, most people seem to get that quickly. Essentially, all interactions (posts, comments, upvotes, subscriptions, etc...) are publicly posted "emails" and lemmy instances are these "email" providers that exchange public data. It could include some basic federation-related ui fundamentals as well, such as listing where and how to find federated content on the site, and a point about how it does/doesn't mix mutually (home.tld/c/community doesn't mix content with otherinstance.tld/c/samecommunity, they're separate). 

### Describe alternatives you've considered

1. Adding a Federated tab on the front page, so it’s Subscribed/Local/Federated/All
2. Adding an "instance" column to the Communities page's table of communities.
3. Adding the community@instance.tld and instance.tld/c/community search syntax.
4. Making other instances' communities visually different from home instance's communities by adding a "federated" tag to them.

### Additional context

1. [Issue for point 1](https://github.com/LemmyNet/lemmy-ui/issues/198)
2. [Issue for point 2](https://github.com/LemmyNet/lemmy-ui/issues/199)
3. [Issue for point 3](https://github.com/LemmyNet/lemmy-ui/issues/200)
4. [Issue for point 4](https://github.com/LemmyNet/lemmy-ui/issues/202)

**Comment by dessalines** [comment]
The onboarding is join.lemmy.ml, which explains that, and that is also linked in several places in the UI.
